app/services/twilio_service.rb

```diff
@@ -12,104 +13,101 @@ class TwilioService
     delivered
     undelivered
     failed
-  ].freeze
+  ).unshift(nil) # why do we need nil in this list, and why must it be first?
```

Comment:
[DUST] i assume you're planning to remove this comment before merging? would prefer not to leave stuff like this around the codebase

Reply:
yes! for sure. i literally don't know the answer but would like to. 😅